software_training/src/trainingNode.cpp

```diff
@@ -0,0 +1,91 @@
+#include "Turtle.h"
```

Comment:
not sure what is going on with this file, it replicates functionality from other files. did you mean to commit the other files or this one? only one of these files needs to be committed